test/MerklePatriciaProof.js

```diff
@@ -0,0 +1,118 @@
+const MerklePatriciaProofMock = artifacts.require("./MerklePatriciaProofMock.sol");
+const accountProofJson =  require("./data/AccountProof.json");
+const storageProofJson =  require("./data/StorageProof.json");
+const ethutil = require ("ethereumjs-util");
+
+contract('MerklePatriciaProof', function(accounts) {
+
+    let merkleMock= null;
+
+    describe('Test Cases for Account proof', async () => {
+        before(async () => {
+            merkleMock = await MerklePatriciaProofMock.new();
+        })
+
+        it('Account Proof Successful', async () => {
+            let encodedValue = '0x'+ethutil.sha3(accountProofJson.value).toString('hex');
+            let proofStatus = await merkleMock.verifyAccount.call(encodedValue, accountProofJson.encodedPath, accountProofJson.rlpParentNodes, accountProofJson.stateRoot);
+            assert.equal(proofStatus,true);
+        })
+
+        it('Account Proof Unsuccessful because encoded path is incorrect', async () => {
+            let encodedValue = '0x'+ethutil.sha3(accountProofJson.value).toString('hex');
+            let proofStatus = await merkleMock.verifyAccount.call(encodedValue, "0x01dB94fdCa0FFeDc40A6965DE97790085d71b413", accountProofJson.rlpParentNodes, accountProofJson.stateRoot);
+            assert.equal(proofStatus,false);
+        })
```

Comment:
it + internal statement should form a sentence.

e.g. it("should not verify account proof when encoded path is incorrect")